CHANGELOG.md

```diff
@@ -9,6 +9,9 @@
   each individual net device will emit metrics under the label `"net_{iface_id}"`.
   E.g. the associated metrics for the endpoint `"/network-interfaces/eth0"` will
   be available under `"net_eth0"` in the metrics json object.
+- [#4063](https://github.com/firecracker-microvm/firecracker/pull/4063):
+  Adds instrumentation based tracing. See [tracing](./docs/tracing.md)
```

Comment:
could probably add that it is source-level instrumentation for those wondering whether it is binary/runtime instrumentation.

Reply:
Updated to

> Adds source-level instrumentation for those ...